ocrd/ocrd/cli/workspace.py

```diff
@@ -43,14 +46,25 @@ def __init__(self, directory, mets_url, mets_basename, automatic_backup):
 @click.option('-d', '--directory', envvar='WORKSPACE_DIR', type=click.Path(file_okay=False), metavar='WORKSPACE_DIR', help='Changes the workspace folder location [default: METS_URL directory or .]"')
 @click.option('-M', '--mets-basename', default=None, help='METS file basename. Deprecated, use --mets/--directory')
 @click.option('-m', '--mets', default=None, help='The path/URL of the METS file [default: WORKSPACE_DIR/mets.xml]', metavar="METS_URL")
+@click.option('-h', '--host', 'mets_server_host', help="host for the server")
+@click.option('-p', '--port', 'mets_server_port',  help="Port for the server", type=int)
+@click.option('-s', '--socket','mets_server_socket',  help="Path to a UNIX socket to be created instead of binding to port")
```

Comment:
> BTW why even allow specifying a host name here? Shouldn't the METS server always be on the same host as the workspace directory? (If you drop the host argument, it must be replaced by the port argument in each is_remote test below. If on the other hand we need to keep it, I still suggest conflating host and port into a single, colon-delimited argument, as you already did for the decorator help string --mets-server-url.)

Yes, it only makes sense to run on the same host, unless there is some NFS magic involved, which is probably unnecessary flexibility.

How about we simplify it even further to just `--mets-server-url`, which can either begin with `http://` in which case it parses that URL for hostname and port or otherwise be interpreted as the path to a socket.

That way, we don't need three options everywehere.

Reply:
46e34bc30 Now only one option `--mets-server-url/-U` throughout (I hope).